items.json

```diff
@@ -14765,9 +14773,9 @@
     "name": "Pilum",

```

Comment:
Since you completely changed the design of the weapon , you need to change the id of the weapon . Add a v2 at the end of the ID . That way it's going to get refunded to thrower that had it